applicationconfig/client/pom.xml

```diff
@@ -92,9 +92,28 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
           <failOnError>true</failOnError>
-          <failOnWarnings>true</failOnWarnings>
         </configuration>
       </plugin>
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>javadoc-warning-fail-java8</id>
+      <activation>
+        <jdk>8</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <configuration>
+              <failOnWarnings>true</failOnWarnings>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
```

Comment:
Invert this - opt out in java-lts, rather than opt-in in jdk 8.

Reply:
Done